utils.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import tensorflow as tf
+def create_time_steps(length):
+    return list(range(-length, 0))
+
+class GetWeights(tf.keras.callbacks.Callback):
```

Comment:
this is a call back as well, should go in callbacks.py

Reply:
u right